src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -305,228 +305,104 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
-		'default': null
-	},
-	'sql.saveAsCsv.textIdentifier': {
-		'type': 'string',
-		'description': localize('sql.saveAsCsv.textIdentifier', "[Optional] Character used for enclosing text fields when saving results as CSV"),
-		'default': '\"'
-	},
-	'sql.saveAsCsv.encoding': {
-		'type': 'string',
-		'description': localize('sql.saveAsCsv.encoding', "[Optional] File encoding used when saving results as CSV"),
-		'default': 'utf-8'
-	},
-	'sql.results.streaming': {
-		'type': 'boolean',
-		'description': localize('sql.results.streaming', "Enable results streaming; contains few minor visual issues"),
-		'default': true
-	},
-	'sql.saveAsXml.formatted': {
-		'type': 'string',
-		'description': localize('sql.saveAsXml.formatted', "[Optional] When true, XML output will be formatted when saving results as XML"),
-		'default': true
-	},
-	'sql.saveAsXml.encoding': {
-		'type': 'string',
-		'description': localize('sql.saveAsXml.encoding', "[Optional] File encoding used when saving results as XML"),
-		'default': 'utf-8'
-	},
-	'sql.copyIncludeHeaders': {
-		'type': 'boolean',
-		'description': localize('sql.copyIncludeHeaders', "[Optional] Configuration options for copying results from the Results View"),
-		'default': false
-	},
-	'sql.copyRemoveNewLine': {
-		'type': 'boolean',
-		'description': localize('sql.copyRemoveNewLine', "[Optional] Configuration options for copying multi-line results from the Results View"),
-		'default': true
-	},
-	'sql.showBatchTime': {
-		'type': 'boolean',
-		'description': localize('sql.showBatchTime', "[Optional] Should execution time be shown for individual batches"),
-		'default': false
-	},
-	'sql.chart.defaultChartType': {
-		'enum': Constants.allChartTypes,
-		'default': Constants.chartTypeHorizontalBar,
-		'description': localize('defaultChartType', "[Optional] the default chart type to use when opening Chart Viewer from a Query Results")
-	},
-	'sql.tabColorMode': {
-		'type': 'string',
-		'enum': [Constants.tabColorModeOff, Constants.tabColorModeBorder, Constants.tabColorModeFill],
-		'enumDescriptions': [
-			localize('tabColorMode.off', "Tab coloring will be disabled"),
-			localize('tabColorMode.border', "The top border of each editor tab will be colored to match the relevant server group"),
-			localize('tabColorMode.fill', "Each editor tab's background color will match the relevant server group"),
-		],
-		'default': Constants.tabColorModeOff,
-		'description': localize('tabColorMode', "Controls how to color tabs based on the server group of their active connection")
-	},
-	'sql.showConnectionInfoInTitle': {
-		'type': 'boolean',
-		'description': localize('showConnectionInfoInTitle', "Controls whether to show the connection info for a tab in the title."),
-		'default': true
-	},
-	'sql.promptToSaveGeneratedFiles': {
-		'type': 'boolean',
-		'default': false,
-		'description': localize('sql.promptToSaveGeneratedFiles', "Prompt to save generated SQL files")
-	},
-	'mssql.intelliSense.enableIntelliSense': {
```

Comment:
I moved these settings to the mssql extension, since it makes more sense for it to be there. There shouldn't be any functional changes.